esmvalcore/_recipe.py

```diff
@@ -489,12 +494,27 @@ def _update_fx_settings(settings, variable, config_user):
         logger.info(msg, 'land/sea fraction weighting', pformat(fx_dict))
 
     for step in ('area_statistics', 'volume_statistics'):
```

Comment:
I think `zonal_statistics` should be added to this list.
```suggestion
    for step in ('area_statistics', 'volume_statistics', 'zonal_statistics'):
```

Reply:
`meridional_statistics` as well?